libs/vkd3d/command.c

```diff
@@ -1305,7 +815,7 @@ static HRESULT STDMETHODCALLTYPE d3d12_fence_Signal(d3d12_fence_iface *iface, UI
     if (fence->timeline_semaphore)
         return d3d12_fence_signal_cpu_timeline_semaphore(fence, value);
     else
-        return d3d12_fence_signal(fence, value, VK_NULL_HANDLE);
+        return d3d12_fence_signal(fence, value);
```

Comment:
Is there any situation where `fence->timeline_semaphore` can be `VK_NULL_HANDLE`?

Looking at `d3d12_fence_init`, we should always be creating a timeline semaphore if `timelineSemaphore` is supported, which is now a required feature.
```
    if (device->device_info.timeline_semaphore_features.timelineSemaphore)
        if (FAILED(hr = d3d12_fence_init_timeline(fence, device, initial_value)))
            return hr;
```

Reply:
Missed it.

Reply:
Should be fixed.